### ISSUE TITLE
Constant rebooting

### ISSUE THREAD

**Issue by kristiyandobrev** [jomjol/AI-on-the-edge-device#1089]
After flashing the firmware, I am experiencing constant rebooting and I can't get the device into a working state 😐. Although it connects to the WIFI network as I can discover it. 

You may see the logs here 👉 https://gist.github.com/kristiyandobrev/e1bf5d410a220e96aeddd5e47bff8af5

My ESP32-CAM board is attached on top of a ESP32-CAM-MB USB Programmer. The power supply is 5v, so I hope power is not an issue here. 

| ![IMG_6024](https://user-images.githubusercontent.com/8437817/192386219-96c771c4-63c0-4b68-9b3b-1d5aa74a739d.jpg)  |  ![IMG_6025](https://user-images.githubusercontent.com/8437817/192386226-0e048357-31f7-434b-99f2-25603961f7f9.jpg)  |  
|---|---|

**Comment by caco3** [comment]
There seems to be  a problem with your camera:
```
23:03:12.308 -> [0;33mW (19842) cam_hal: Failed to get the frame on time![0m
23:03:12.308 -> [0;31mE (19842) main: esp_camera_fb_get: Camera Capture Failed[0m
23:03:12.308 -> Camera cannot be initialzed. System will reboot.
23:03:12.308 -> Reboot triggert by Software (5s).
23:03:12.345 -> [0;32mI (19862) server_ota: Reboot in 5sec[0m
23:03:12.345 -> Reboot in 5sec
```

Can you remove and attach it again? Notice the black lever must be opened for this!

**Comment by kristiyandobrev** [comment]
Thanks for the suggestion @caco3. No change in behaviour after doing that multiple times...

**Comment by caco3** [comment]
hmm..

Which software version are you using?

maybe @jomjol has an idea.

**Comment by kristiyandobrev** [comment]
> hmm..
> 
> Which software version are you using?
> 
> maybe @jomjol has an idea.

Latest.

**Comment by caco3** [comment]
It could be that that camera is not supported. The naming on the ribbon cable looks not familiar to me.
I also have a camera which is not supported!

**Comment by kristiyandobrev** [comment]
Yep, the camera seems to be different. The one I use appears to be `8225N` and the one that is supported according to the codebase is `OV2640`.

**Comment by friedpa** [comment]
Had same problem long time ago, use an OV2640 modell

**Comment by kristiyandobrev** [comment]
Alright guys, thanks ! I will get a hold of the `OV2640` model and report my findings here.
